OptionalCommand/Jackalope/InitDoctrineDbalCommand.php

```diff
@@ -38,7 +39,7 @@ protected function configure()
 
         $this
             ->setName('doctrine:phpcr:init:dbal')
-            ->addOption('session', null, InputOption::VALUE_OPTIONAL, 'The session to use for this command', 'default')
+            ->addOption('session', null, InputOption::VALUE_OPTIONAL, 'The session to use for this command')
```

Comment:
this will likely cause breakage for anyone that doesn't configure an admin session